etcdmgmt/etcd-mgmt.go

```diff
@@ -142,9 +143,20 @@ func isETCDStartNeeded() (bool, int) {
 			start = true
 			return start, pid
 		}
-
 		if exist := utils.CheckProcessExist(pid); exist == true {
-			start = false
+			hostname, err := os.Hostname()
+			if err != nil {
+				log.Fatal("Could not able to get hostname")
```

Comment:
This is exactly what I commented in your patch that you would not need to return. log.Fatal does an exit() which means the entire application will terminate.